vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
+
+{{< pagebreak >}}
+
+# Analysis Considerations Related to Multiple Analysis Datasets
+
+## Core Variables
+
+Core variables are those that are represented across all/most analysis datasets.
+
+| **Variable Name**   | **Variable Description**              |
+|---------------------|---------------------------------------|
+| STUDYID             | Study Identifier                      |
+| USUBJID             | Unique Subject Identifier             |
+| SUBJID              | Subject Identifier for the Study      |
+| SITEID              | Study Site Identifier                 |
+| SITEGR1             | Pooled Site Group 1                   |
+| TRTSDT              | Date of First Exposure to Treatment   |
+| TRTEDT              | Date of Last Exposure to Treatment    |
+| AGE                 | Age                                   |
+| AGEGR1              | Pooled Age Group 1                    |
+| AGEGR1N             | Pooled Age Group 1 (N)                |
+| RACE                | Race                                  |
+| RACEN               | Race (N)                              |
+| SEX                 | Sex                                   |
+| SAFFL               | Safety Population Flag                |
+| ITTFL               | Intent-To-Treat Population Flag       |
+| EFFFL               | Efficacy Population Flag              |
+| COMP24FL            | Completers of Week 24 Population Flag |
+| DSRAEFL             | Discontinued due to AE?               |
+
+## Treatment Variables
+
+ARM versus TRT01P
+
+```         
+  Are the values of ARM equivalent in meaning to values of TRT01P?
+
+    Yes.
+```
+
+ACTARM versus TRT01A
+
+```         
+  If TRT01A is used, then are the values of ACTARM equivalent to values of TRT01A?
+
+    Not applicable - ACTARM is not used.
+```
+
+Use of ADaM Treatment Variables in Analysis
+
+```         
+  Are both planned and actual treatment variables used in analysis?
+
+    Yes. Planned treatment variables are used for study population and efficacy 
+    analyses, whilst actual treatment variables are used for the safety analysis.
+    All subjects received the treatment arm to which they were randomised and so
+    the planned treatment is equivalent to the actual treatment for all subjects.
+```
+
+Use of ADaM Treatment Grouping Variables in Analysis
+
+```         
+  Are both planned and actual treatment grouping variables used in analysis?
+
+    Not applicable - treatment grouping variables are not used.
+```
+
+## Use of Visit Windowing, Unscheduled Visits, and Record Selection
+
+Was windowing used in one or more analysis datasets?
+
+```         
+  Yes
+```
+
+Were unscheduled visits used in any analyses?
+
+```         
+  Yes
+```
+
+## Imputation/Derivation Methods
+
+For ASTDT in ADAE, this date was converted to numeric SAS date from AE.AESTDTC. If the day component is missing, a value of '01' is used. If both the month and day are missing no imputation is performed. See define.xml.
```

Comment:
Did we convert it to a numeric SAS date while using R?  Maybe that part could be dropped??

Reply:
Yes, we did.  As this was apart of the specs, we also had to cover it in the program regardless of whether or not any dates needed imputation or not.  The function `admiral::derive_vars_dtm()` handles this imputation for us.